tensorflow/tools/graph_transforms/fold_batch_norms.cc

```diff
@@ -73,7 +73,7 @@ Status FoldBatchNorms(const GraphDef& input_graph_def,
 
         // Make sure all the inputs really are vectors, with as many entries as
         // there are columns in the weights.
-        const int weights_cols_index = conv_node.op() == "Conv2D" ? 3 : 1;
+        const int weights_cols_index = conv_node.op() == "Conv2D" ? 3 : (conv_node.op() == "DepthwiseConv2dNative" ? 2 : 1);
```

Comment:
This line is too long, you might need to break it to 2 lines.

Reply:
Thanks @mingxingtan , I test it with mobilenet_v1_1.0_224 on CPU(E5-2650v2) and saw 10% performance improvement compared to original model(without folding bn). Graph topology is simplized, it may perform better for inference on mobile devices.

Reply:
Nice, after you break this long line into two lines, I will LGTM.

Reply:
I modified this line and pushed it.